compiler-rt/lib/mpk_untrusted/alloc_site_handler.h

```diff
@@ -193,6 +206,27 @@ class AllocSiteHandler {
     }
   }
 
+  void storePidKey(pid_t threadID, PKeyInfo pkey) {
```

Comment:
can we document what these APIs are for?

Reply:
Added documentation notes.